composer.json

```
@@ -20,7 +20,9 @@
     "require": {
         "php": "^7.2",
         "cocur/slugify": "^1.4 || ^2.0 || ^3.0 || ^4.0",
-        "sonata-project/doctrine-extensions": "^1.1",
+        "sonata-project/exporter": "^1.11 || ^2.0",
+        "sonata-project/form-extensions": "0.1",
+        "sonata-project/twig-extensions": "0.1",
```

@wbloszyk please rephrase, I don't understand. I was suggesting this:

```suggestion
        "sonata-project/form-extensions": "0.1@dev",
        "sonata-project/twig-extensions": "0.1@dev",
```

because the build fails and I think it would fix it. The alternate solution would be to tag `0.1.0` for both packages, which I can do.